bowtie2/2.4.4/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# FROM defines the base docker image to start from. This command has to come first in the file
+FROM ubuntu:xenial
+
+ARG SAMTOOLSVER=1.11
+ARG BOWTIE2VER=2.4.4
+
+# metadata (there are a few other labels you can add, these are optional but preferred!)
+LABEL base.image="ubuntu:xenial"
+LABEL version="1"
+LABEL software="Bowtie2,\n Samtools"
+LABEL software.versions="2.4.4,\n 1.10"
+LABEL description="Bowtie2: Genome assembler using a reference and mapping\n Samtools: a set of tools for interacting with and reformatting sequence data"
+LABEL website="http://bowtie-bio.sourceforge.net/bowtie2/manual.shtml\n https://www.htslib.org/"
+LABEL maintainer="Holly Halstead"
+LABEL maintainer.email="holly.halstead@doh.wa.gov"
+
+ENV ZIP=bowtie2-${BOWTIE2VER}-linux-x86_64.zip
+ENV URL=https://github.com/BenLangmead/bowtie2/releases/download/v${BOWTIE2VER}/
+ENV FOLDER=bowtie2-${BOWTIE2VER}
+ENV DST=/home/dock/bin
```

Comment:
I don't believe the $DST directory is used anywhere? Might be good to remove this line as well as the command below on line 24 to make the directory `mkdir -p $DST && \`

Doesn't hurt to keep it around though. simply an empty directory

Reply:
Looks like all of these variables are unused, at least in the docker image building process. Maybe I'm missing the reasoning these are included

Up to you if you want to remove them!